Dashboard-README.md

```diff
@@ -0,0 +1,54 @@
+# Dashboard for visualizing conference summaries
+## Github Page URL: https://loveyourself999.github.io/conference-dashboard/
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+***
+## Overall Idea
+### Build a dashboard application that visualize the youtube summaries to nevigate video content
+1. Create an information dashboard using Next.js and Recharts.(https://ably.com/blog/informational-dashboard-with-nextjs-and-recharts#creating-the-charts-component)
+2. Use D3 library for data visualization when developing the UI(https://d3js.org)
+3. Fetch data from the summaries data collected by Github Action
+4. Deploy dashboard app to GitHub Pages with GitHub Actions(https://pages.github.com)
+***
+## Building Steps
+### To build a next.js app:
+Make sure you installed npm by
+~~~Bash
+npm -v
+
+// if not installed run
+npm install -g npm@latest
+~~~
+Create a next.js file using command "npx create-next-app + your file name"
+~~~Bash
+npx create-next-app conference-dashboard
+cd conference-dashboard
+~~~
+Run the app at [http://localhost:3000](http://localhost:3000)with your browser to see the result.
+~~~bash
+npm run dev
+~~~
+
+Editing the page by modifying `page.js`. The page auto-updates as you edit the file.
+
+### To deploy app to Github Pages with Github Action
+First create a repository for the app, push the code.
+
+Enable GitHub Pages:
+1. Go to your repository's Settings tab
+2. Click "Pages" in the sidebar
+3. Under "Build and Deployment", select "GitHub Actions" as the source:
+<img width="1314" alt="Screen Shot 2024-04-26 at 4 57 09 PM" src="https://github.com/LoveYourself999/conference-dashboard/assets/54390035/c9999bf4-1c6c-45ee-b1e6-52d6e624b079">
```

Comment:
please update the URL

Reply:
Should I remove the image?

Reply:
please add the image to the repo if it is needed to explain the overall design.